packages/default-plugins/package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "@joplin/default-plugins",
+	"version": "2.13.0",
```

Comment:
Also please update setupNewRelease.ts to automatically update this version number